README.md

```diff
@@ -41,13 +41,62 @@
 </a>
 </p>
 
-This is an experimental web client for Jellyfin based on Vue.js. We welcome all contributions and pull requests! If you have a larger feature in mind please open an issue so we can discuss the implementation before you start.
+This is an alternative client for Jellyfin based on Vue.js. It's not feature complete and it's constantly evolving.
 
-<p align="center">
-    <a href="https://jf-vue.pages.dev/">View demo</a>
-</p>
+We welcome all contributions and pull requests! If you have a larger feature in mind please open a discussion or chat with us in Matrix so we can discuss the implementation before you start.
```

Comment:
Maybe add a link to matrix?

Reply:
@thornbill The badge is already a link. Do you have any other suggestion so I can make it more clear that's whrere users must click?

Reply:
I think if there is a link in a badge that's fine.